src/components/Chat/Chat.js

```diff
@@ -11,13 +11,13 @@ let socket;
 
 const Chat = ({ location }) => {
   const [user, setUser] = useState("");
-  const emptyShoutout = { 
+  const emptyShoutout = {
```

Comment:
I'm seeing some merge issues in this file. the emptyShoutout object should take this shape:   
```
const emptyShoutout = {
    author: "",
    color: "",
    comments: [],
    recipient: "",
    text: "",
    variant: "shoutout",
  }
```